apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -30,6 +37,21 @@ export const ConnectWalletButton: FC<
 }) => {
   const { t } = useTranslation();
   const [isOpen, toggle] = useReducer(state => !state, false);
+  const { addNotification } = useNotificationContext();
+
+  const onCopyAddress = useCallback(() => {
+    addNotification(
+      {
+        type: NotificationType.success,
+        title: t(translations.copyAddress),
+        content: '',
+        dismissible: true,
+        id: Math.floor(Math.random() * 1000),
```

Comment:
We use it in multiple places, pls extract it to a util method (e.g. `generateRandomId`).

Reply:
used nanoid package here instead